src/AspNet.Security.OAuth.Yahoo/YahooAuthenticationOptions.cs

```diff
@@ -27,10 +27,10 @@ public YahooAuthenticationOptions()
 
             ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "guid");
             ClaimActions.MapJsonKey(ClaimTypes.Name, "nickname");
-            ClaimActions.MapJsonKey("urn:yahoo:familyname", "familyName");
-            ClaimActions.MapJsonKey("urn:yahoo:givenname", "givenName");
-            ClaimActions.MapJsonKey("urn:yahoo:profile", "profileUrl");
-            ClaimActions.MapJsonKey("urn:yahoo:profileimage", "imageUrl");
+            ClaimActions.MapJsonKey(YahooClaimTypes.FamilyName, "familyName");
+            ClaimActions.MapJsonKey(YahooClaimTypes.GivenName, "givenName");
```

Comment:
Is there a reason a lot of these are using custom claim types and some of the built-in ones, like `ClaimTypes.GivenName`, `ClaimTypes.Surname`, etc.?

Reply:
Most likely an oversight (most of the aspnet-contrib providers have been written by different contributors, who may not be super familiar with all the built-in claim names).

FWIW, I try to avoid using the "legacy" WS-Fed claims as much as possible (because they're super long and make cookies/tokens bigger) but the MSFT OAuth2 providers use them when it's possible so we use them too for consistency.

Reply:
Right, so as long as there's no transformations for them anywhere, it doesn't really matter? Or, it might even be better to use custom claim types because of the smaller size?

Reply:
Let's go with the built-in `ClaimTypes` when we have a good constant for these claims. Consistency always wins, I guess.

Reply:
I didn't want to change any claim types in this PR, as it would be a breaking change. Should we open an issue for later?

Reply:
Breaking changes are kinda acceptable as we're still in RC. But yeah, separate PR if you prefer :+1: